### ISSUE TITLE
Antivirus addon

### ISSUE THREAD

**Issue by GijsVK94** [amidaware/tacticalrmm#1161]
i know there is a oppertiunity to deploy endpoint antivirus software but this is still managed in a different dashboard.
to keep everything in one place it would be great to see another icon in the dashboard of TRRM for the virusscanner

**Comment by silversword411** [comment]
What av? You mean just query windows security center to see it's opinion?

**Comment by GijsVK94** [comment]
to deploy something like bitdefender.
what i mean is that if i install the trrm agent on a system the endpoint av will be installed automaticaly.
when this is done a icon will be seen (like the Patching pending, pending action, reboot icons) so we also see the notification in the trrm dashboard and not in a different dashboard like gravity zone

**Comment by silversword411** [comment]
With 50+ av products out in the market there's not a lot TRMM is going to be able to do to support everything.

You can deploy whatever you want now (including av) using the existing scripting capabilities in TRMM

Integration of additional indicators for agents (av or otherwise) will probably be part of whatever https://github.com/amidaware/tacticalrmm/issues/653

tagging system gets implemented.

I can foresee at some point in the future maybe some official av integration with one or several av vendors possibly as part of an extension process similar to other RMM systems.

I'm tempted to close this issue as it's kind of many issues in one, but will leave open for now and let devs decide.

**Comment by joeldeteves** [comment]
I think what the poster is referring to is the AV integration that exists in major RMM products like Kaseya and Labtec:

- Sophos
- ESET
- Microsoft
- BitDefender (yuck)
- etc

It's not necessary, sure, but It might expand the interest in TRMM to a wider audience if there was integration with the usual suspects.

@wh1te909: I would certainly consider paying for this as an optional "premium" feature similar to Code Signed agents.

**Comment by silversword411** [comment]
Yes, I'm sure this will be a long term extension/monetization mechanism :)
